test/test_core.py

```diff
@@ -6179,6 +6180,20 @@ def test_sigalrm(self):
   def test_signals(self):
     self.do_core_test(test_file('test_signals.c'))
 
+  @parameterized({
+    'sigint': (['sigint', signal.SIGINT, 128 + signal.SIGINT, True]),
+    'sigabrt': (['sigabrt', signal.SIGABRT, 7, False])
+  })
+  def test_sig_default(self, suffix, signal, exit_code, assert_identical):
```

Comment:
Can you give this a longer name?  `sig` also means `signature` in a lot of places in emscripten.  How about `test_sigaction_default` or `test_default_signal_handlers`?